models/rke-machine.cattle.io.digitaloceanmachinetemplate.js

```diff
@@ -1,6 +1,6 @@
 import MachineTemplate from './rke-machine.cattle.io.machinetemplate';
 
-export default class DigialoceanMachineTemplate extends MachineTemplate {
+export default class DigitaloceanMachineTemplate extends MachineTemplate {
```

Comment:
api?

Reply:
Modifying an export - commenting to circle back for a closer look later on.

Reply:
With that said, it is a default export and not a named export, so I think this change is safe..

Reply:
GitHub tells me that it only offered one other instance of this string:
https://cs.github.com/?scopeName=All+repos&scope=&q=DigialoceanMachineTemplate
https://github.com/bart-rijnders/dashboard/blob/21c71f84cc81342a86d116a7ccef88cfee450027/models/rke-machine.cattle.io.digitaloceanmachinetemplate.js#L3
That's because https://github.com/bart-rijnders/dashboard -- This branch is 622 commits behind rancher:master.